### DISCUSSION TITLE
Tempo Traces To Logs with custom query generation capabilites

### DISCUSSION THREAD

**Discussion by gurudatta-carbon** [grafana/grafana#56817] [Data Sources]
Tempo Datasource's [Trace To Logs](https://grafana.com/docs/grafana/latest/explore/trace-integration/#trace-to-logs), currently maps Trace resource attributes to mapped datasource (eg: loki).

If logs are in format of json, and has only few labels, then having more customization will be helpful.
Example I have an opentelemetry collector that pushes Json logs to Loki, traces to Tempo.
Tempo is configured with Loki for "Traces To Logs".

**Generated Query:** 
{ resources_service_name = "serviceName" } | ="6346e053b6aa4eec12b882ce00eb0ea0"
**Note:** there is mapping of,  service.name=resources_service_name in tempo datasource

**Workable Query for my use case:**
{exporter="OTLP"} | json | traceid="6346e053b6aa4eec12b882ce00eb0ea0"
**Note:** exporter attribute is not available, but is it possible to hardcode it, and add json parser

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
